src/components/converter/ComboInput.js

```diff
@@ -1,75 +1,92 @@
-import React, { useState } from 'react'
+import React, { useState, useCallback } from 'react'
 import styled from 'styled-components'
+import 'styled-components/macro'
 import { animated, useSpring } from 'react-spring'
-import Token from './Token'
 import Arrow from './assets/arrow.svg'
 
 const AnimDiv = animated.div
+const noop = () => {}
+
 function ComboInput({
   inputValue,
-  options,
-  selectedOption,
-  onSelect,
-  onChangeInput,
-  setEditMode,
+  options = [''],
+  selectedOption = 0,
+  onBlur = noop,
+  onChange = noop,
+  onFocus = noop,
+  onSelect = noop,
+  placeholder,
 }) {
   const [opened, setOpened] = useState(false)
   const { openProgress } = useSpring({
     openProgress: opened ? 1 : 0,
   })
```

Comment:
As we were discussing offline, I think we should try to accelerate this spring! 🏎💨  :smile: